app/models/client.rb

```diff
@@ -45,7 +45,7 @@ class Client < ApplicationRecord
 
   belongs_to :vita_partner, optional: true
   has_many :documents, dependent: :destroy
-  has_one :intake, dependent: :destroy
+  has_one :intake, inverse_of: :client, dependent: :destroy
```

Comment:
@alaroia-cfa I learned what inverse_of does: When you are creating an object _through_ another object (setting dependents_attributes on an intake while creating a client), if you use inverse_of on the relationship it will allow you to create the children objects through the parent. If not, you can't do it.